src/pages/settings/Profile/ProfilePage.js

```diff
@@ -213,120 +149,70 @@ class ProfilePage extends Component {
     }
 
     render() {
-        const pronounsList = _.map(this.props.translate('pronouns'), (value, key) => ({
-            label: value,
-            value: `${CONST.PRONOUNS.PREFIX}${key}`,
-        }));
         const currentUserDetails = this.props.currentUserPersonalDetails || {};
-        const pronounsPickerValue = this.state.hasSelfSelectedPronouns ? CONST.PRONOUNS.SELF_SELECT : this.pronouns;
-
+        const profileSettingsOptions = [
+            {
+                description: this.props.translate('displayNamePage.headerTitle'),
+                title: `${lodashGet(currentUserDetails, 'firstName', '')} ${lodashGet(currentUserDetails, 'lastName', '')}`,
+                pageRoute: ROUTES.SETTINGS_DISPLAY_NAME,
+            },
+            {
+                description: this.props.translate('pronounsPage.pronouns'),
+                title: this.getPronouns(),
+                pageRoute: ROUTES.SETTINGS_PRONOUNS,
+            },
+            {
+                description: this.props.translate('timezonePage.timezone'),
+                title: `${lodashGet(currentUserDetails, 'timezone.selected', '')}`,
+                pageRoute: ROUTES.SETTINGS_TIMEZONE,
+            },
+        ];
         return (
-            <ScreenWrapper>
+            <ScrollView>
```

Comment:
Was this needed for some style fixes or something?

Reply:
I wanted to make the content scrollable. I realised that when I inspect the App on web, I can't access all menu items on the ProfilePage if the dev tools are rendered at the bottom of the window. Doesn't it make sense?
<img width="1449" alt="Screenshot 2022-12-06 at 14 04 25" src="https://user-images.githubusercontent.com/18078393/205907953-dca150ff-a4bc-4b12-b0b8-e97adf114e89.png">

Reply:
Ooh well i definitely think it's good you caught this - about it not being able to scroll before 😅 (in case the user has a very short screen I guess?)

But I thinkkk you should put the `ScrollVew` as a sibling of `HeaderWithCloseButton` like we have here: https://github.com/Expensify/App/blob/fa19a2d5e6f27dbc0f72ead48afddcd85e1d20b5/src/pages/ReportDetailsPage.js#L110-L116

Right?

Reply:
you're right, thanks